haskell-pkgs.nix

```diff
@@ -1,18 +1,20 @@
 let
   # Fetch the latest haskell.nix and import its default.nix
-  haskellNix = import (builtins.fetchTarball{
-    url = "https://github.com/input-output-hk/haskell.nix/archive/f6663a8449f5e4a7393aa24601600c8f6e352c97.tar.gz";
-  }) {};
+  haskellNix = import
+    (builtins.fetchTarball {
+      url = "https://github.com/input-output-hk/haskell.nix/archive/1b9b05beed75a1be98405501fbb33ac1f080069e.tar.gz";
```

Comment:
In a fututre we might want to consider using niv (https://input-output-hk.github.io/haskell.nix/tutorials/getting-started/#niv) but for now I've left this as is